dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -1963,14 +2022,16 @@ def _get_num_zeros(
         :type subset_properties: dict
         :return: None
         """
+        if not self._greater_than_64_bit:
+            df_series = pl.from_pandas(df_series)
```

Comment:
Is this supposed to be converting `pd.Series` to `np.ndarray`? I read `if not self._greater_than_64_bit` as "if this isn't a ndarray". Correct me if I am not following the logic correctly.